VideoWeb/VideoWeb/ClientApp/src/app/shared/toast/room-closing/room-closing-toast.component.scss

```diff
@@ -0,0 +1,44 @@
+.call {
```

Comment:
maybe worth revisiting the vhtoastr scss
maybe we could add something to the '.ts' like 
`styleUrls: ['../vhtoast.scss', ....]` so we dont have to remember to update 2 places?

Reply:
Done